### ISSUE TITLE
Improve the user experience when minor/major version of service changes

### ISSUE THREAD

**Issue by wvangeit** [ITISFoundation/osparc-issues#1536]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### User Story

Can we please improve the user experience when a service is being upgraded to a new minor/major version.
It seems in this case the automatic update buttons are not available in the study screen, and the user has to, as far as I can see, delete the existing service from the study, and reconnect the connection, which can be rather cumbersome if this has to be done for several studies.

I understand the reasoning for making sure very well realizing the behavior changes, but it seems this could still be solved by adding an extra warning box, before letting users update to a new minor/major version.

### Definition of Done

User can more easily, more automated, upgrade to a new minor/major version of a service

**Comment by elisabettai** [comment]
Maybe @odeimaiz has also some ideas on how to improve this. 

With Esra we once thought about a button/checkbox to "automatically" copy the data to a new version (basically what now the user has to do by end).

What @wvangeit, suggests is rather an extra "confirmation" pop-up which basically would make the user agreeing to update, "at their own risk", right? And then we change version in the DB as we currently do.

**Comment by wvangeit** [comment]
Yeah, if we really want to warn the user, we could let them type some 'I AGREE' or something in a box

**Comment by odeimaiz** [comment]
Also, if there is a major upgrade, the internal data structure might have changed making your old data useless or ignored. Not sure if upgrade at your own risk is safe enough...

That's why we have the integration-version and display-version. One deals with all the above and the later is the one used for marketing purposes.
